jobs/views.py

```diff
@@ -276,6 +376,12 @@ def get_form_kwargs(self):
             kwargs['initial'] = {'email': self.request.user.email}
         return kwargs
 
+    def get_context_data(self, **kwargs):
+        ctx = super().get_context_data()
+        ctx.update(kwargs)
+        ctx['needs_preview'] = not self.request.user.is_staff
```

Comment:
Two things:

1. We have a "Job Board Admin" group for reviewers and we will switch to use it. See #768.
2. I don't think we should change the usage of the job board for reviewers. It would be good to remove other `needs_` instances, too.